src/Configuration.cs

```diff
@@ -637,6 +677,91 @@ public void DrawConfig(float x_size, float y_size) {
+                                    { nameof(slideCol), new Vector4(0f, 0f, 0f, 0.4f) },
+                                    { nameof(ctComboUsed), new Vector4(0.431f, 0.431f, 0.431f, 1f) },
+                                    { nameof(ctComboActive), new Vector4(1f, 1f, 1f, 1f) },
+                                    { nameof(BarBackCol), new Vector4(0.376f, 0.376f, 0.376f, 0.667f) },
+                                    { nameof(BarFrontCol), new Vector4(0.9f, 0.9f, 0.9f, 1f) },
+                                    { nameof(BarOgcdCol), new Vector4(1f, 1f, 1f, 1f) },
+                                    { nameof(BarAnLockCol), new Vector4(0.334f, 0.334f, 0.334f, 0.667f) },
+                                    { nameof(BarclipCol), new Vector4(1f, 0f, 0f, 0.667f) },
+                                    { nameof(CastBarTextColor), new Vector3(1f, 1f, 1f) },
+                                    { nameof(BarHasGradient), false },
+                                    { nameof(BarGradMode), 2 },
+                                    { nameof(BarBgGradMode), 3 },
+                                    { nameof(BarGradientMul), 0.175f },
+                                    { nameof(BarBgGradientMul), 0.175f },
+                                });
+                                showColorConfirmation = false;
+                                ImGui.CloseCurrentPopup();
+                            }
+                            ImGui.SameLine();
+                            if (ImGui.Button("Use DelvUI Inspired Colors")) {
+                                ResetColors(new Dictionary<string, object> {
+                                    { nameof(clipCol), new Vector4(0.9882353f, 0.32941177f, 0.0f, 0.8f) },
+                                    { nameof(abcCol), new Vector4(0.9882353f, 0.8235294f, 0.0f, 0.8f) },
+                                    { nameof(ClipTextColor), new Vector4(0.9f, 0.9f, 0.9f, 1.0f) },
+                                    { nameof(ClipBackColor), new Vector4(0.9882353f, 0.32941177f, 0.0f, 1.0f) },
+                                    { nameof(abcTextColor), new Vector4(0.0f, 0.0f, 0.0f, 1.0f) },
+                                    { nameof(abcBackColor), new Vector4(1.0f, 0.7f, 0.0f, 1.0f) },
+                                    { nameof(backCol), new Vector4(0.3764706f, 0.3764706f, 0.3764706f, 0.8f) },
+                                    { nameof(backColBorder), new Vector4(0.0f, 0.0f, 0.0f, 1.0f) },
+                                    { nameof(frontCol), new Vector4(0.0f, 0.63529414f, 0.9882353f, 1.0f) },
+                                    { nameof(ogcdCol), new Vector4(1.0f, 1.0f, 1.0f, 1.0f) },
+                                    { nameof(anLockCol), new Vector4(1.0f, 1.0f, 1.0f, 0.26666668f) },
+                                    { nameof(BarBackColBorder), new Vector4(0.0f, 0.0f, 0.0f, 1.0f) },
+                                    { nameof(slideCol), new Vector4(0.6745098f, 0.0f, 0.9882353f, 0.8f) },
```

Comment:
DelvUI uses red, but that looks horrible if you have the background red from the clip.  So that lavender color seemed to be the best alternative that worked with the other two colors.  I’ll change it in the config.

Reply:
Done.